### DISCUSSION TITLE
Share button

### DISCUSSION THREAD

**Discussion by ckarpinski** [fancyapps/ui#185] [General]
Does Fancybox 4 support a share button in the toolbar like Fancybox 3 had? I have tried adding "share" to the options the way you would for version 3 but that does not seem to work

**Comment by fancyapps** [comment]
Hi,

First, I believe that share buttons are a thing of the past. Try googling smth like "do people use social network sharing buttons", you will find many old articles praising share buttons, but recent articles will question their effectiveness. Also, see this article - https://vwo.com/success-stories/taloon/ Personally, I have never clicked on any share button and I find them completely useless. Even worse, I automatically dislike websites that are bloated with share toolbars and buttons. 

So far, you are the first person to ask about Share toolbar. If more people were interested, I could create optional plugin for that. As alternative, you can create your own buttons in the toolbar, check out this demo with twitter and facebook buttons - https://fancyapps.com/playground/1mG 
(Twitter share popup will not load in the demo, possibly because playground uses an iframed page)

**Comment by ckarpinski** [comment]
Hi, The context I used fancybox in includes users who have asked for the ability to share their own images from our site. It was a requested featured.

I will checkout the playground and see if I can figure it out.

Thanks for the reply

________________________________

**Reply by fancyapps** [reply]
So, are your users okay with a couple of sharing buttons, or do you really need a sharing toolbar?

**Comment by ckarpinski** [comment]
I think either would be fine, but I am not sure what the difference would be so I cant say for sure.

________________________________
